core/vm/context.go

```diff
@@ -14,7 +14,7 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
 
-package core
+package vm
```

Comment:
Just flagging that this change will make pulling in future changes more difficult

Reply:
I agree!

Reply:
At least you could wait until i finish 1.9 merge :P

Reply:
Is there a way to just "link" things from the original package?